src/payment/strategies/masterpass/masterpass-payment-strategy.ts

```diff
@@ -120,6 +120,8 @@ export default class MasterpassPaymentStrategy extends PaymentStrategy {
             currency: storeConfig.currency.code,
             cartId: checkout.cart.id,
             suppressShippingAddress: false,
+            callbackUrl: window.location.origin +
+                '/checkout.php?action=set_external_checkout&provider=masterpass&gateway=stripe&origin=checkout',
```

Comment:
Can this string be moved to a constant and be interpolated here vs. concatenated? We are also copy/pasting the same code 3x is there a way that we can make this DRY?